runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -68,7 +69,9 @@
 
 	// Notifier properties
 	private static final Color FLASH_COLOR = new Color(255, 0, 0, 70);
-	private static final int FLASH_DURATION = 2000;
+	private static final int MINIMUM_FLASH_DURATION_MILLIS = 2000;
+	private static final int MINIMUM_FLASH_DURATION_TICKS = 33; // 33 = 2000 / 60 rounded down
```

Comment:
This doesn't look correct. Per documentation for `client.getMouseIdleTicks()` and `client.getKeyboardIdleTicks()`, those values are measured in client ticks, not game ticks, which are defined in Constants to be 20ms in length. Additionally, this should be `private static final int MINIMUM_FLASH_DURATION_TICKS = MINIMUM_FLASH_DURATION_MILLIS / 60;` (or with 60 replaced with the appropriate constant reference) so that the comment isn't needed.

Reply:
Ah... good spot, yes you’re right. I’ll fix that later thanks!